packages/googlechrome.vm/tools/chocolateyinstall.ps1

```diff
@@ -20,7 +20,33 @@ try {
         fileType        = 'MSI'
         silentArgs      = "/quiet /norestart /l*v `"$($env:TEMP)\$($env:chocolateyPackageName).$($env:chocolateyPackageVersion).MsiInstall.log`""
     }
-    Install-ChocolateyInstallPackage @packageArgs
+    Install-ChocolateyInstallPackage @packageArgs -ErrorAction SilentlyContinue
+
+    $chromePath = "${env:ProgramFiles}\Google\Chrome\Application\chrome.exe"
+    VM-Assert-Path $chromePath
```

Comment:
[nitpick] to make it more compact:
```suggestion
    Install-ChocolateyInstallPackage @packageArgs -ErrorAction SilentlyContinue
    VM-Assert-Path "%ProgramFiles%\Google\Chrome\Application\chrome.exe"
```

Reply:
Interestingly enough, that seems to fail?
![image](https://github.com/mandiant/VM-Packages/assets/35854880/4397cbe7-072d-41e2-a703-05a00d5bf270)